generate-test-vector.py

```diff
@@ -1007,8 +1024,8 @@ def generate_all_inputs_test():
         generate_multiple_output_tests() +
         generate_labeled_output_tests() +
         generate_multiple_outputs_with_labels_tests() +
-        generate_change_tests() +
-        generate_all_inputs_test(),
+        generate_change_tests(),
+        # generate_all_inputs_test(),
```

Comment:
I see the best is commented, does it pass on the reference implementation?

Reply:
IIRC, it wasn't and I was more focused on fixing up the test structure first. Also, the BIP has changed a bit since when you first started writing that test, such as hybrid pubkeys are no longer allowed (meaning they would be skipped in the inputs).